app/lib/filters/official_documents_filter.rb

```diff
@@ -0,0 +1,7 @@
+module Filters
+  class OfficialDocumentsFilter < RadioFilterForMultipleFields
+    def filter_hashes
+      Filters::OfficialDocumentsHashes.new.call
```

Comment:
You have used inheritance to provide `filter_hashes` here, and composition for the facet classes. It might be worth using composition for both, but can always do this later!